discussion/app/views/fragments/comment.scala.html

```diff
@@ -43,7 +43,7 @@
                     </a>
                 }
                 <div class="d-comment__timestamp">
-                    <a href="@Configuration.discussion.url/comment-permalink/@comment.id" class="d-comment__timestamp block-time">
+                    <a href="@Configuration.discussion.url/comment-permalink/@comment.id" class="d-comment__timestamp block-time" data-link-name="Comment permalink">
```

Comment:
I'd check with the data team, but we have enabled outbound tracking in omniture.